packages/app/src/features/spark-rewards/components/claim-all-panel/ClaimAllPanel.tsx

```diff
@@ -182,3 +163,74 @@ function MultipleTokensWithoutPrice({ tokens }: { tokens: ActiveReward[] }) {
 function TokenIconOrSymbol({ token, iconClassName }: { token: Token; iconClassName?: string }) {
   return isTokenImageAvailable(token.symbol) ? <TokenIcon token={token} className={iconClassName} /> : token.symbol
 }
+
+interface ClaimableTokensWithPriceSubPanelProps {
+  usdSum: NormalizedUnitNumber
+  claimableTokensWithPrice: ActiveReward[]
+  claimableTokensWithoutPrice: ActiveReward[]
+}
+
+function ClaimableTokensWithPriceSubPanel({
+  usdSum,
+  claimableTokensWithPrice,
+  claimableTokensWithoutPrice,
+}: ClaimableTokensWithPriceSubPanelProps) {
+  return (
+    <SubPanel>
+      <div className="flex items-center gap-1">
+        <div className="typography-label-2 text-tertiary">Total to claim</div>
+        <Info>Sum of all rewards available to claim in USD.</Info>
+      </div>
+      <div className="flex items-center gap-2">
+        <div className="typography-heading-2 text-primary-inverse">{USD_MOCK_TOKEN.formatUSD(usdSum)}</div>
+        <IconStack size="m" items={claimableTokensWithPrice.map(({ token }) => token)} />
+      </div>
+      <div className="flex flex-col gap-2">
+        {claimableTokensWithoutPrice.map(({ token, amountToClaim }) => (
+          <div key={token.symbol} className="typography-label-2 flex items-center gap-1 text-tertiary">
+            +{token.format(amountToClaim, { style: 'compact' })}{' '}
+            <TokenIconOrSymbol token={token} iconClassName="size-4" />
+          </div>
+        ))}
+      </div>
+    </SubPanel>
+  )
+}
+
+interface PendingRewardsSubPanelProps {
+  usdSum: NormalizedUnitNumber
+}
+
+function PendingRewardsSubPanel({ usdSum }: PendingRewardsSubPanelProps) {
+  return (
+    <SubPanel>
+      <div className="flex items-center gap-1">
+        <div className="typography-label-2 text-tertiary">Total to claim</div>
+        <Info>You have nothing to claim at the moment.</Info>
+      </div>
+      <div className="typography-heading-2 text-primary-inverse">{USD_MOCK_TOKEN.formatUSD(usdSum)}</div>
```

Comment:
Can be just
```suggestion
      <div className="typography-heading-2 text-primary-inverse">{USD_MOCK_TOKEN.formatUSD(0)}</div>
```
I guess?